testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_predective_queries.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+
+from CBLClient.Document import Document
+from CBLClient.PredictiveQueries import PredictiveQueries
+from libraries.data import doc_generators
+from libraries.testkit import cluster
+from keywords.utils import deep_dict_compare
+
+
+@pytest.mark.sanity
+@pytest.mark.listener
+@pytest.mark.parametrize(
+    'doc_generator_type',
+    [
+        ('four_k'),
+        ('simple_user'),
+        ('complex_doc'),
+        ('simple')
+    ]
+)
+def test_predictiveQueries_basicInputOutput(params_from_base_test_setup, doc_generator_type):
```

Comment:
Jim has create few more doc type would be great if we can embed his data, as they don't have random values.

Reply:
which doc type? Did he added in testserver app? I added four data type. Complex data has covered most of them which also generates random values when data is generated.